Changelog.md

```diff
@@ -6,6 +6,7 @@ Language Features:
 Compiler Features:
  * Build System: LLL is not built anymore by default. Must configure it with CMake as `-DLLL=ON`.
  * Code generator: Do not perform redundant double cleanup on unsigned integers when loading from calldata.
+ * Optimiser: Replace `push_back` with `emplace_back`.
```

Comment:
I think this change is rather internal and don't need a Changelog entry.

Reply:
Alright, will do!